res/skins/QMLDemo/EffectSlot.qml

```diff
@@ -0,0 +1,196 @@
+import "." as Skin
+import Mixxx 0.1 as Mixxx
+import QtQuick 2.12
+import "Theme"
+
+Item {
+    id: root
+
+    property var slot: Mixxx.EffectsManager.getEffectSlot(1, unitNumber, effectNumber)
```

Comment:
Didn't work when I tried, then I read that it is not possible to use uncreatable types as property types. But according to this bug report it should be fixed since Qt 5.8: https://bugreports.qt.io/browse/QTBUG-36752 I'll check again.